unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/RPC/Transports/WebSocketTransport/WebSocketTransport.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using rpc_csharp.transport;
+
+namespace RPC.Transports
+{
+    // TODO: fix issues when websocket reconnect!
```

Comment:
Which issues are we having? The Unity-side doesn't care about reconnecting. The Desktop connecting flow it's like:

- Kernel tries to connect to Unity
- Kernel tries to connect to Unity
- Kernel tries to connect to Unity
- Unity starts listening
- Kernel connects

Reply:
I can't recall... I think there were actually no issue there... but I do recall you telling me that you were going to refactor the websocket service 🙄😝